src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -957,15 +960,18 @@ static ucs_status_t uct_ib_mlx5_devx_md_open(struct ibv_device *ibv_device,
         goto err_free;
     }
 
-    status = uct_ib_mlx5_devx_query_lag(md, &lag_state);
+    status = uct_ib_mlx5_devx_query_lag(md, &lag_state, &port_select_mode);
     if (status != UCS_OK) {
-        dev->lag_level = 0;
+        dev->lag_level   = 0;
+        port_select_mode = UCT_IB_LAG_RESERVED_MODE;
```

Comment:
usage of RESERVED value seems weird

Reply:
@evgeny-leksikov As it's described in ```LAG CONTEXT Field Descriptions``` in PRM: ```Other values are reserved```. If it can't get the lag mode through query function ```status != UCS_OK```, it needs to assign an unmeaningful value to ```dev->port_select_mode```. ```RESERVED``` plays the role of ```unmeaningful value```.

Reply:
so it should have different name in UCT, for example UCT_IB_LAG_INVALID_MODE

Reply:
@evgeny-leksikov Done. rename ```UCT_IB_LAG_RESERVED_MODE``` to be ```UCT_IB_LAG_INVALID_MODE```